lib/AST/ASTMangler.cpp

```diff
@@ -2797,7 +2803,8 @@ bool ASTMangler::appendGenericSignature(GenericSignature sig,
 }
 
 void ASTMangler::appendRequirement(const Requirement &reqt,
-                                   GenericSignature sig) {
+                                   GenericSignature sig,
+                                   bool baseIsProtocolSelf) {
```

Comment:
Could you name this in some way that makes it clear it only applies to the first type?

Reply:
Named and documented.